test/controller.go

```diff
@@ -80,6 +81,8 @@ type Data struct {
 	ResolutionRequests      []*resolutionv1alpha1.ResolutionRequest
 	ExpectedCloudEventCount int
 	VerificationPolicies    []*v1alpha1.VerificationPolicy
+	StatefulSets            []*v1.StatefulSet
```

Comment:
It might make sense to create a different test data struct, since the existing seedtestdata function doesn't handle statefulsets or nodes (https://github.com/tektoncd/pipeline/blob/main/test/controller.go#L169)